randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1102,7 +1086,9 @@ Extra - map_name: rm_a5c27
       Any of the following:
           Ice Beam
           Damage Boost (Beginner) and Normal Damage ≥ 50
-          Gravity Suit and Speed Booster
+          All of the following:
+              Gravity Suit
+              Hi-Jump Boots or Space Jump or Speed Booster or Can Use Spider Ball
```

Comment:
Underwater zips are possible here too.
So in the big any condition, you probably want to add

**ALL**
- Morph
- Zip trick (Beginner)